AlertToViber/run.csx

```diff
@@ -47,10 +47,10 @@ public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, IQuery
         if (matchedString != null && matchedString != string.Empty)
         {
             // Parse and Decode and Join if the string type is UTF-8
-            if (matchedString.StartsWith("=?utf-8?B?") || matchedString.StartsWith("=?UTF-8?B?"))
+            if (matchedString.StartsWith("=?"))
             {
-                log.Info("Subject of this incoming email contains UTF-8 chars");
-                matchedString = ParseAndDecodeAndJoinUTF8String(matchedString);

```

Comment:
I changed the name of the method because the method adapted to not only UTF-8 but also JIS encoding.

Reply:
OK. I got it.